services/client/src/main/java/org/collectionspace/services/client/IQueryManager.java

```diff
@@ -51,6 +51,7 @@ public interface IQueryManager {
 	final static String SEARCH_QUALIFIER_OR = SEARCH_TERM_SEPARATOR + "OR" + SEARCH_TERM_SEPARATOR;
 	final static String DEFAULT_SELECT_CLAUSE = "SELECT * FROM ";
 	final static String CSID_QUERY_PARAM = "csid";
+	String TAG_QUERY_PARAM = "tag";
```

Comment:
Should put `final static` here, like the other constants. (The conventional order is `static final`, but  🤷)

I'm thinking we should name the query parameter `servicetag`, in case we add tags to individual records or something in the future. It's a bit unclear what "tag" applies to when you look at the URL. Using `tag` in the service and application config is fine, since it's clear in context what's being tagged.